deploy/deploy-k8s.yml

```diff
@@ -8,3 +8,4 @@
     kube_version: "v1.12.1"
     kubeconfig_localhost: true
     local_volumes_enabled: true
+    kube_feature_gates: ["CSIPersistentVolume=true","MountPropagation=true","VolumeSnapshotDataSource=true","KubeletPluginsWatcher=true","CSINodeInfo=true","CSIDriverRegistry=true"]
```

Comment:
I'm having trouble finding some of these in the docs: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

If we're installing 1.12, then we shouldn't need:
- CSIPersistentVolume is enabled by default since 1.10
- MountPropagation enabled since 1.10 (and now GA)
- KubeletPluginsWatcher enabled since 1.12

Don't see docs for:
- CSIDriverRegistry
- CSINodeInfo
- VolumeSnapshotDataSource

Reply:
@JohnStrunk Thanks for the review.
i was following this setup document from CSI https://kubernetes-csi.github.io/docs/Setup.html

when i started with implementing this feature, these flags are mentioned to be enabled,
now setup doc is changed. i will update This PR